README.md

```diff
@@ -1,14 +1,11 @@
-# JMDict
+# JMDict-go
```

Comment:
Nitpick but I would prefer jmdict-go here

Reply:
Weird maybe I'm over looking it but looking at the branch on github https://github.com/xythh/jmdict-go it shows it like that.

Reply:
Hmm? I mean the capitalization

Reply:
Understood :+1: , I just changed both of them to lowercase.